src/interactive.js

```diff
@@ -150,9 +154,15 @@ module.exports = function (manifest, manifestPath, ugly) {
 
     function promptFieldValue(fieldName) {
         let value = '';
+        let currentValuePrompt = manifest[fieldName] ? '('.grey + manifest[fieldName].grey + ') '.grey : "";
+        let verb = manifest[fieldName] ? 'Edit' : 'Enter';
```

Comment:
Does this also work correctly if you try to add/edit fields such as `icons.48`, `developer.name`, `activities.dhis.href` or `my.custom.field.name` ?